### DISCUSSION TITLE
SPARQL Editor - eclipse plugin

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#3915] [General]
Is there someting like an SPARQL Editor - eclipse plugin that can be used to run queries against localhost?
Thanks Fredy

**Comment by abrokenjester** [comment]
I'm not aware of anything that is a plugin. The closest I can think of is that you can run the RDF4J Console directly from Eclipse, so you'd have a command line terminal in your Eclipse console window.

**Comment by kenwenzel** [comment]
KOMMA's RDF editor components include a simple SPARQL view, but it is mainly intended for querying model sets and not for arbitrary SPARQL endpoints.

**Comment by frensjan** [comment]
I've resorted to a simple HTML file with https://github.com/TriplyDB/Yasgui embedded in it. With a little bit of JavaScript, you can configure the default endpoint.

**Reply by naturzukunft** [reply]
if there is time, i will take a look on that. Yesterday i created a local testcase to execute queries ;-)
I like the tool! I use it here: https://yasgui.triply.cc/
